client/stylesheet.less

```diff
@@ -55,12 +68,35 @@ body {
 	display: flex;
 	justify-content: space-between;
 	margin-top: 20px;
+	max-width: 800px;
+	min-width: 400px;
 	.column {
 		flex: 1;
 		min-height: @column-min-height;
 	}
 }
 
+.row-label-column {
+	flex: 0.5;
+	.row-labels {
+		display: flex;
+		flex-direction: column;
+		color: white;
+		height: 100%;
+		display: flex;
+		text-align: right;
```

Comment:
You have two display properties in this block. Also, I don't think text-align is doing anything here.